### PR TITLE
Fix empty ChassisConfig problem in YangParseTreeTest

### PR DESCRIPTION
**Pull request by mhwang408**
@Yi-Tseng Empty ChassisConfig results in low coverage because
some codes in on_set_functor will never be hit

### CODE REVIEW

**Comment by Yi-Tseng** [comment]
Please rebase to new master or merger the master to resolve the conflict

File: stratum/hal/lib/common/yang_parse_tree_test.cc [hunk 1]
```diff
@@ -109,15 +109,17 @@ class YangParseTreeTest : public ::testing::Test {
   }
 
   // A proxy for YangParseTree::AddSubtreeInterface().
-  void AddSubtreeInterface(const std::string& name) {
+  void AddSubtreeInterface(const std::string& name,
+                           SingletonPort* singleton_p) {
```
Comment (Yi-Tseng) [inline stratum/hal/lib/common/yang_parse_tree_test.cc]: You can use `SingletimePort* singleton_p = nullptr` for default value, so you don't need to modify line 176, 575, 707, ... etc,

File: stratum/hal/lib/common/yang_parse_tree_test.cc [hunk 2]
```diff
@@ -109,15 +109,17 @@ class YangParseTreeTest : public ::testing::Test {
...
     absl::WriterMutexLock l(&parse_tree_.root_access_lock_);
-
-    // Add one singleton port.
     SingletonPort singleton;
-    singleton.set_name(name);
-    singleton.set_node(kInterface1NodeId);
-    singleton.set_id(kInterface1PortId);
-    singleton.set_speed_bps(kTwentyFiveGigBps);
+    SingletonPort* singleton_ptr = (singleton_p == nullptr)? &singleton \
```
Comment (Yi-Tseng) [inline stratum/hal/lib/common/yang_parse_tree_test.cc]: Add space before `?`
Also please break the line from `=`
```
SingletonPort* singleton_ptr =
        (singleton_p == nullptr) ? &singleton : singleton_p;
```

Comment (mhwang408) [reply stratum/hal/lib/common/yang_parse_tree_test.cc]: I encounter a problem....So I delete the branch and redo works on a new branch. 
I've sent a new pull request.

**Pull request closed without merging**